packages/data-driver-postgres/package.json

```diff
@@ -28,10 +28,15 @@
 		"access": "public"
 	},
 	"devDependencies": {
+		"@directus/data": "workspace:*",
```

Comment:
I think we've once discussed that we should use a separate package (or use `@directus/types` 🤔) for shared things to prevent circular dependencies between `@directus/data` <-> `@directus/data-driver-` in general.

Reply:
true 🤔 so like this [this](https://github.com/directus/data#how-the-packages-are-related) actually

Reply:
There's should be no direct dependency from `@directus/data` to the individual drivers. `@directus/data` never imports / installs the drivers themselves, so from a dependency perspective I don't think it's circular 🤔 

We can always split it up later if we run into unforeseen issues tho! 😄

Reply:
ohh right.. that the drivers aren't imported by `data` `is different from what we had it in the prototype hence a little confusion. To see it at a glance I created an updated version of the package diagram above. So now the structure is like [this](https://github.com/directus/directus/tree/basic-pg-driver/packages/data#dependencies)

Reply:
> There's should be no direct dependency from `@directus/data` to the individual drivers. `@directus/data` never imports / installs the drivers themselves, so from a dependency perspective I don't think it's circular 🤔
> 
> We can always split it up later if we run into unforeseen issues tho! 😄

Makes sense for me if we can keep it like this 👍